src/leappto/cli.py

```diff
@@ -756,11 +745,24 @@ def print_migrate_info(text):
 
         # if el7 then use systemd
         if machine_src.installation.os.version.startswith('7'):
-            result = mc.start_container('centos:7',
-                                        '/usr/lib/systemd/systemd --system',
-                                        tcp_mapping)
+            opts = [
+                # Required for docker 1.13
+                "--cap-add", "SYS_ADMIN",
+                # Implemented in default command
+                #"-v", "/sys/fs/cgroup:/sys/fs/cgroup:ro",
+                "--tmpfs", "/run", 
+                "--tmpfs", "/tmp",
+                "-e", "container=docker"
+                
+            ]
+
+            result = mc.start_container('centos/systemd',
+                                        #'/usr/lib/systemd/systemd --system',
+                                        '/sbin/init',
```

Comment:
If the commented out command isn't needed any more, it should be removed entirely. However, it may still be worth commenting that `/sbin/init` kicks everything off correctly, so it's being called on purpose rather than as a holdover from CentOS 6.